DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1893,6 +1896,42 @@ private void BroadcastGame()
             if (ProgramConstants.IsInGame && broadcastChannel.Users.Count > 500)
                 return;
 
+            string gameBroadcastingString = BuildGameBroadcastingString();
+
+            // @TODO: Sending ctcp for broadcasting may not be needed now, if its only informing cncnet lobby
+            broadcastChannel.SendCTCPMessage(gameBroadcastingString, QueuedMessageType.SYSTEM_MESSAGE, 20);
+
+            // @TODO: Perhaps we should only update the topic when something has changed?
+            connectionManager.SetChannelTopic(channel, gameBroadcastingString);
```

Comment:
See the comment above about generating a ton of waste traffic. We would need to rethink when to send that too. TODOs are correct, they just need to be done. 😛